packages/ember-glimmer/lib/modifiers/action.js

```diff
@@ -194,7 +209,13 @@ export default class ActionModifierManager {
   update(modifier, element, args, dom, dynamicScope) {
     let { positional } = args;
 
-    modifier.actionName = positional.at(1).value();
+    let actionNameRef = positional.at(1);
+
+    if (actionNameRef[INVOKE]) {
+      modifier.actionName = actionNameRef;
+    } else {
+      modifier.actionName = actionNameRef.value();
+    }
     modifier.eventName = modifier.getEventName();
 
     // Not sure if this is needed? If we mutate the actionState is that good enough?
```

Comment:
I do not see the point of the lines below.  Seems a but odd, I think we can delete them.